app/templates/components/forms/wizard/basic-details-step.hbs

```diff
@@ -315,30 +315,11 @@
     <div class="field">
       <h4>{{t 'Add Payment Details'}}</h4>
       <div class="two fields">
-        <div class="field">
-          <label class="required">{{t 'Select country'}}</label>
-          <UiDropdown
-            @class="search selection"
-            @selected={{this.data.event.paymentCountry}}
-            @forceSelection={{false}}
-            @fullTextSearch={{true}}>
-            <Input
-              @type="hidden"
-              @autocomplete="no"
-              @id="payment_country"
-              @value={{this.data.event.paymentCountry}} />
-            <i class="dropdown icon"></i>
-            <div class="default text">{{t 'Select country'}}</div>
-            <div class="menu">
-              {{#each this.paymentCountries as |paymentCountry|}}
-                <div class="item" data-value="{{paymentCountry.name}}">
-                  <i class="{{lowercase paymentCountry.code}} flag"></i>
-                  {{paymentCountry.name}}
-                </div>
-              {{/each}}
-            </div>
-          </UiDropdown>
-        </div>
+      <CountryDropdown
+        @required={{true}}
+        @paymentCountries={{true}}
+        @placeholder={{'Select country'}}
```

Comment:
t missing

Reply:
used in countryDropdown only

Reply:
It has to be used with static strings, not dynamic

Reply:
so I should like this ??
```handlebars
<label class={{if @required 'required'}}>{{@placeholder}}</label>
```
and 
```handlebars
@placeholder={{t 'Select Country'}}
```